src/models/flinkStatement.ts

```diff
@@ -157,6 +186,12 @@ export class FlinkStatementTreeItem extends TreeItem {
         ["Detail", resource.status.detail],
       ],
     );
+
+    this.command = {
```

Comment:
Make viewing the SQL statement as a read-only document the default action when clicking on a flink statement.